detector/aws_elb_invalid_subnet.go

```diff
@@ -10,22 +10,20 @@ import (
 
 type AwsELBInvalidSubnetDetector struct {
 	*Detector
-	IssueType  string
-	TargetType string
-	Target     string
-	DeepCheck  bool
-	subnets    map[string]bool
+	subnets map[string]bool
 }
 
 func (d *Detector) CreateAwsELBInvalidSubnetDetector() *AwsELBInvalidSubnetDetector {
-	return &AwsELBInvalidSubnetDetector{
-		Detector:   d,
-		IssueType:  issue.ERROR,
-		TargetType: "resource",
-		Target:     "aws_elb",
-		DeepCheck:  true,
-		subnets:    map[string]bool{},
+	nd := &AwsELBInvalidSubnetDetector{
+		Detector: d,
+		subnets:  map[string]bool{},
 	}
+	nd.Name = "aws_elb_invalid_subnet"
+	nd.IssueType = issue.ERROR
+	nd.TargetType = "resource"
+	nd.Target = "aws_elb"
```

Comment:
__[gometalinter]__
* [goconst] 3 other occurrence(s) of "aws_elb" found in: aws_elb_duplicate_name.go:23:14 aws_elb_invalid_instance.go:24:14 aws_elb_invalid_security_group.go:24:14

<!-- Comment by [SideCI](https://sideci.com) -->